### ISSUE TITLE
Datetime.datetime throws TypeError: 'Datetime' object is not callable

### ISSUE THREAD

**Issue by JonasMedu** [lk-geimfari/mimesis#1139]
# Bug report
As far as i can tell, the function "datetime" is not callable via the Schema creation.

According to the Api reference https://mimesis.name/api.html#mimesis.Datetime.datetime and the example in the getting started section i hoped i can create a schema as shown in Whats Wrong.

## What's wrong

```python
from mimesis.locales import Locale
from mimesis.schema import Field, Schema

_ = Field(locale=Locale.EN)
schema = Schema(schema=lambda: {
    "year": 2021,
    "uid": _("uuid"),
    "name": _("text.word"),
    "datetime": _("datetime"),
    "timestamp": _("timestamp", posix=False),
    "version": _("version", pre_release=True),
    "owner": _("text.word"),
})
list_of_dicts = schema.create(iterations=5)
```

Traceback:
`TypeError: 'Datetime' object is not callable`

## How is that should be

A dictionary `list_of_dicts ` should be created.

## System information

Win10, Anaconda Python installation

**Comment by lk-geimfari** [comment]
@JonasMedu Can you try this, please: `_("datetime.datetime")`?

**Comment by JonasMedu** [comment]
@lk-geimfari `_("datetime.datetime")` works nicely.
ref:
```python
from mimesis.locales import Locale
from mimesis.schema import Field, Schema


_ = Field(Locale.EN)

schema = Schema(schema=lambda: {
    "year": 2021,
    "datetime": _("datetime.datetime", start=2021, end=2021),
    "name": _("text.word"),
    "company": _("company"),
    "target_value": _("integer_number", start=15200.0, end=1e10),
    "timestamp": _("timestamp", posix=False),
    "version": _("version", pre_release=True),
    "owner": _("stock_name"),
})
example = schema.create(iterations=10000)
```

**Comment by lk-geimfari** [comment]
It is better to leave this issue open, because I did not fix the bug yet (since it is a bug). I'll close this issue on my own.

**Comment by lk-geimfari** [comment]
@ngnpope I see that you've contributed a lot to mimesis lately, so maybe you'll find this bug interesting. I've fixed this issue in #1139, but my solution is too ugly (and too dumb) to merge it to master. Maybe you can solve this problem gracefully than I did.
